lib/topology_docker_openswitch/openswitch.py

```diff
@@ -206,16 +202,16 @@ def main():
 
 PROCESS_LOG = """
 #!/bin/bash
-ovs-vsctl list Daemon >> /tmp/logs
-echo "Coredump -->" >> /tmp/logs
-coredumpctl gdb >> /tmp/logs
-echo "All the running processes:" >> /tmp/logs
-ps -aef >> /tmp/logs
+ovs-vsctl list Daemon >> /var/topology/logs
```

Comment:
Please add a header for this section also, so that the output of ``/var/topology/logs`` is identified properly, like:

    Output of: ovs-vsctl list Daemon >> /var/topology/logs